### ISSUE TITLE
'dotnet-mssql' and 'dotnet' containers: running test-project locally fails consistently

### ISSUE THREAD

**Issue by dnastrain** [microsoft/vscode-dev-containers#1190]
- VSCode Version: 1.62.3 (system setup)
- Local OS Version: Windows_NT x64 10.0.22000
- Reproduces in: Remote - Containers (test), and potentially CodeSpaces
- Name of Dev Container Definition with Issue: _dotnet-mssql_ and _dotnet_

Steps to Reproduce:

1. Open either _dotnet-mssql_, or _dotnet_, container folder in VSCode
2. Click on "Reopen in Container"
3. Right-click `test-project` folder and select "Open in Integrated Terminal"
4. Within the terminal, run: `dotnet restore&& dotnet build&& dotnet run`
5. _Notice:_  both _restore_ & _build_ succeed, and _run_ fails with:

```
It was not possible to find any compatible framework version
The framework 'Microsoft.AspNetCore.App', version '5.0.0' (x64) was not found.
  - The following frameworks were found:
      6.0.0 at [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The specified framework can be found at:
  - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=5.0.0&arch=x64&rid=ubuntu.20.04-x64
```

**Proposed Solution:**
1. Update test-project/aspnetapp.csproj _in both projects_, to use `TargetFramework` of "netcoreapp**6.0**" over "netcoreapp**5.0**"

**Comment by dnastrain** [comment]
cc:@Chuxel for context

**Comment by Chuxel** [comment]
Yeah we should update the test projects since the default was updated to 6.0.
